src/SemConv/TraceAttributes.php

```diff
@@ -495,7 +674,7 @@ interface TraceAttributes
     public const DB_CASSANDRA_TABLE = 'db.cassandra.table';
 
     /**
-     * The connection string used to connect to the database. It is recommended to remove embedded credentials.
+     * Deprecated, use `server.address`, `server.port` attributes instead.
```

Comment:
I'll commit some tweaks to the template once I have some time later today, but I'm unsure if this is due to schema changes, or if it may cause BC-breaks... Either way, I'll try to keep the current behaviour as a fallback.

Reply:
The most likely explanation is that semconv has evolved and our template hasn't. Java's [just committed](https://github.com/open-telemetry/semantic-conventions-java/commit/9202fd6cdc1b21e05558d17b6f13608345a3f80b) 1.25.0 and checking their template, there looks to be a newer way to test for deprecation: https://github.com/open-telemetry/semantic-conventions-java/blob/main/buildscripts/templates/SemanticAttributes.java.j2#L79

Reply:
I've updated the template again, using the same `is_deprecated` filter (and thus hopefully reproducing the same behavior). Can you please check if this one's OK, or should I still change something?

Reply:
Looking much better now! I think it's more correct to use the `deprecated` field as you have, otherwise it's just duplicating the brief that's already there.